tests/components/Drawer.test.js

```diff
@@ -28,4 +48,29 @@ describe('Drawer Component', () => {
 
         expect(asFragment()).toMatchSnapshot();
     });
+
+    it('should disable/enable body scroll when drawer is toggled', () => {
```

Comment:
I think we should assert `enableBodyScroll` and `disableBodyScroll` are called with the expected/target element.

Reply:
Yap, done! 👍